doc/integrator/ogc_api.rst

```diff
@@ -0,0 +1,53 @@
+.. _integrator_ogc_api:
+
+OGC API Features
+----------------
+
+The OGC API is a new standard that defines modular API following REST standard to access spatial data.
+The current implementation is only on the server side and is based on the OGC API - Features standard and only works with the `items` request using a bbox filter.
+
+It is currently supported by both MapServer and QGIS Server.
+
+The following new environment variables should be added to the docker-compose-lib.yaml file for the 'MapServer' service:
+
+.. prompt:: bash::
+
+    environment:
+        ...
+        - MAPSERVER_BASE_PATH=/mapserv_proxy
+        - OGCAPI_HTML_TEMPLATE_DIRECTORY=/usr/local/share/mapserver/ogcapi/templates/html-bootstrap4/
```

Comment:
This should be added to the `docker-compose-lib.yaml` provided by c2cgeoportal!

Reply:
So I can ignore it in the doc ?

Reply:
yes